MantidQt/CustomInterfaces/src/Reflectometry/QtReflEventView.cpp

```diff
@@ -80,13 +80,8 @@ std::string QtReflEventView::getTimeSlicingValues() const {
 * @return :: Time slicing type
 */
 std::string QtReflEventView::getTimeSlicingType() const {
-  std::map<SliceType, std::string> sliceTypeStrMap = {
-      {Uniform, "Uniform"},
-      {UniformEven, "UniformEven"},
-      {Custom, "Custom"},
-      {LogValue, "LogValue"}};
 
-  return sliceTypeStrMap[m_sliceType];
+  return m_sliceTypeStrMap.at(m_sliceType);
```

Comment:
Why did you switch to `std::map::at`? It's a relatively expensive check that in this case serves no purpose.